scripts/mockapi.js

```diff
@@ -167,6 +167,7 @@ export {
   fetchDynamicStockIndexData,
   callMockBlogAPI,
   getTrendingNews,
+  getHostUrl,
```

Comment:
@ravverma  this is something I have merged with marketcommentary as well in main . So you can remove it from here to avoid conflict before merging

Reply:
ok sure. 👍

Reply:
done